dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -11,7 +11,7 @@ def index
   end
 
   def k5_public_map_index
-    @workshops = @workshops.where(
+    @workshops = Pd::Workshop.where(
```

Comment:
Oh, never mind. I already did that :)

This [serializes](https://github.com/code-dot-org/code-dot-org/blob/bd8f28007a2ae38647fcbc9d2a13d2d84d9ed956/dashboard/app/controllers/api/v1/pd/workshops_controller.rb#L19-L19) with the [WorkshopK5MapSerializer](https://github.com/code-dot-org/code-dot-org/blob/7d452d6bb463a0c6368e7384d3da27166bc98361/dashboard/app/serializers/api/v1/pd/workshop_k5_map_serializer.rb) that only displays `:id, :location_name, :location_address, :sessions`